addons/hr_expense/static/src/components/expense_dashboard.js

```diff
@@ -22,16 +22,7 @@ export class ExpenseDashboard extends Component {
     }
 
     renderMonetaryField(value, currency_id) {
-        value = value.toFixed(2);
-        const currency = session.get_currency(currency_id);
-        if (currency) {
-            if (currency.position === "after") {
-                value += currency.symbol;
-            } else {
-                value = currency.symbol + value;
-            }
-        }
-        return value;
+        return formatMonetary(value, { currencyId: currency_id});;
```

Comment:
I just wonder is there any changes that `currency_id` might be empty :grey_question:

Reply:
I'm pretty sure that in the context of this view it will be set :smile: